js/main.js

```diff
@@ -1,21 +1,121 @@
-//gen randon int
-const getRandomInt = (min, max) => {
-  if (min > 0 && max > 0 && min < max) {
-    const randomInt = min + Math.random() * (max + 1 - min);
-    return Math.floor(randomInt);
+const TITLE = [
```

Comment:
```suggestion
const TITLES = [
```
все массивы должны быть во множественном числе (нужно исправить везде)